src/SFA.DAS.EmployerApprenticeshipsService.Application/Queries/GetAccountTransferRole/GetAccountTransferRoleHandler.cs

```diff
@@ -0,0 +1,49 @@
+using System.Data;
+using System.Data.Entity;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using AutoMapper.QueryableExtensions;
+using MediatR;
+using SFA.DAS.EAS.Application.Data;
+using SFA.DAS.EAS.Application.Dtos;
+using SFA.DAS.EAS.Application.Validation;
+using SFA.DAS.EAS.Domain.Models.TransferConnections;
+using SFA.DAS.HashingService;
+
+namespace SFA.DAS.EAS.Application.Queries.GetAccountTransferRole
+{
+    public class GetAccountTransferRoleHandler : IAsyncRequestHandler<GetAccountTransferRoleQuery, GetAccountTransferRoleResponse>
```

Comment:
Do you think it's worth adding a unit test for this class?

Reply:
Done